### PR TITLE
Fix bug and UI as per story

### PR DESCRIPTION
**Pull request by harryul**

### CODE REVIEW

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js [hunk 1]
```diff
@@ -0,0 +1,2483 @@
...
+  var DOCUMENT = _DOCUMENT;
+  var MUTATION_OBSERVER = _MUTATION_OBSERVER;
+  var PERFORMANCE = _PERFORMANCE;
+  var IS_BROWSER = !!WINDOW.document;
+  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === 'function' && typeof DOCUMENT.createElement === 'function';
+  var IS_IE = ~userAgent.indexOf('MSIE') || ~userAgent.indexOf('Trident/');
+
+  var NAMESPACE_IDENTIFIER = '___FONT_AWESOME___';
+  var UNITS_IN_GRID = 16;
+  var DEFAULT_FAMILY_PREFIX = 'fa';
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:167]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var DEFAULT_FAMILY_PREFIX = "fa";
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js [hunk 2]
```diff
@@ -0,0 +1,2483 @@
...
+  var _DOCUMENT = {};
+  var _MUTATION_OBSERVER = null;
+  var _PERFORMANCE = {
+    mark: noop,
+    measure: noop
+  };
+
+  try {
+    if (typeof window !== 'undefined') _WINDOW = window;
+    if (typeof document !== 'undefined') _DOCUMENT = document;
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:148]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
    if (typeof document !== 'undefined') {_DOCUMENT = document;}
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+  } catch (e) {}
+
+  var _ref = _WINDOW.navigator || {},
+      _ref$userAgent = _ref.userAgent,
+      userAgent = _ref$userAgent === void 0 ? '' : _ref$userAgent;
+
+  var WINDOW = _WINDOW;
+  var DOCUMENT = _DOCUMENT;
+  var IS_BROWSER = !!WINDOW.document;
+  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === 'function' && typeof DOCUMENT.createElement === 'function';
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:23]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === "function" && typeof DOCUMENT.createElement === 'function';
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js [hunk 4]
```diff
@@ -0,0 +1,2483 @@
...
+  var _ref = _WINDOW.navigator || {},
+      _ref$userAgent = _ref.userAgent,
+      userAgent = _ref$userAgent === void 0 ? '' : _ref$userAgent;
+
+  var WINDOW = _WINDOW;
+  var DOCUMENT = _DOCUMENT;
+  var MUTATION_OBSERVER = _MUTATION_OBSERVER;
+  var PERFORMANCE = _PERFORMANCE;
+  var IS_BROWSER = !!WINDOW.document;
+  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === 'function' && typeof DOCUMENT.createElement === 'function';
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:162]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === "function" && typeof DOCUMENT.createElement === 'function';
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+        _defineProperty(target, key, source[key]);
+      });
+    }
+
+    return target;
+  }
+
+  var w = WINDOW || {};
+  if (!w[NAMESPACE_IDENTIFIER]) w[NAMESPACE_IDENTIFIER] = {};
+  if (!w[NAMESPACE_IDENTIFIER].styles) w[NAMESPACE_IDENTIFIER].styles = {};
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:81]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  if (!w[NAMESPACE_IDENTIFIER].styles) {w[NAMESPACE_IDENTIFIER].styles = {};}
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+    /**
+     * Font Awesome 4 used the prefix of `fa` for all icons. With the introduction
+     * of new styles we needed to differentiate between them. Prefix `fa` is now an alias
+     * for `fas` so we'll easy the upgrade process for our users by automatically defining
+     * this as well.
+     */
+
+
+    if (prefix === 'fas') {
+      defineIcons('fa', icons);
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:117]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
      defineIcons("fa", icons);
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js [hunk 7]
```diff
@@ -0,0 +1,998 @@
...
+
+    if (conflictsCount > 0) {
+      console.info("%cConflict".concat(conflictsCount > 1 ? 's' : '', " found:"), 'color: darkred; font-size: large');
+      var data = {};
+
+      for (var _key in nodesTested.conflict) {
+        var item = nodesTested.conflict[_key];
+        data[_key] = {
+          'tagName': item.tagName,
+          'src/href': item.src || item.href || 'n/a',
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js:120]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
          "src/href": item.src || item.href || 'n/a',
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+      _ref$userAgent = _ref.userAgent,
+      userAgent = _ref$userAgent === void 0 ? '' : _ref$userAgent;
+
+  var WINDOW = _WINDOW;
+  var DOCUMENT = _DOCUMENT;
+  var IS_BROWSER = !!WINDOW.document;
+  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === 'function' && typeof DOCUMENT.createElement === 'function';
+  var IS_IE = ~userAgent.indexOf('MSIE') || ~userAgent.indexOf('Trident/');
+
+  var NAMESPACE_IDENTIFIER = '___FONT_AWESOME___';
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js:29]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var NAMESPACE_IDENTIFIER = "___FONT_AWESOME___";
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js [hunk 9]
```diff
@@ -0,0 +1,2483 @@
...
+  var _MUTATION_OBSERVER = null;
+  var _PERFORMANCE = {
+    mark: noop,
+    measure: noop
+  };
+
+  try {
+    if (typeof window !== 'undefined') _WINDOW = window;
+    if (typeof document !== 'undefined') _DOCUMENT = document;
+    if (typeof MutationObserver !== 'undefined') _MUTATION_OBSERVER = MutationObserver;
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:149]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
    if (typeof MutationObserver !== 'undefined') {_MUTATION_OBSERVER = MutationObserver;}
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js (hunk 1 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:167]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js (hunk 2 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:148]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js (hunk 3 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:23]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js (hunk 4 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:162]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js (hunk 5 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:81]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js (hunk 6 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:117]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js (hunk 7 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js:120]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js (hunk 8 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js:29]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js (hunk 9 above)
Comment (harryul) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:149]: This is third party libraries. Please review this @vietvo361

File: src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml [hunk 10]
```diff
@@ -33,39 +37,53 @@
                 <div class="col l-4">
                     <div class="item patient-item1">
                         <div>
-                            <img src="~/Image/no-avatar.jpg" alt="Avatar" class="patient-img" />
+                            @if (item.AvatarLink == "")
```
Comment (vietvo361) [inline src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml:40]: Should use `string.IsNullOrEmpty()`

File: src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml [hunk 11]
```diff
@@ -33,39 +37,53 @@
...
                             </div>
                             <p class="contact-info">@item.PhoneNumber</p>
                         </div>
                         <div>
                             <div class="contact-icon">
                                 <i class="fas fa-envelope"></i>
                             </div>
-                            <p class="contact-info">@item.Email</p>
+                            @{
+                                if (item.Email != null)
```
Comment (vietvo361) [inline src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml:80]: Should use `string.IsNullOrEmpty()`

File: src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml [hunk 12]
```diff
@@ -33,39 +37,53 @@
...
+                                {
+                                    string[] result = item.Email.Split("@");
+                                    <p class="email-link">@result[0]</p>
+                                    <p class="contact-info">@@@result[1]</p>
```
Comment (vietvo361) [inline src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml:84]: Why this has 3 '@' ?

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js [hunk 13]
```diff
@@ -0,0 +1,998 @@
...
+    var noConflictsCount = Object.keys(nodesTested.noConflict).length;
+
+    if (noConflictsCount > 0) {
+      console.info("%cNo conflict".concat(noConflictsCount > 1 ? 's' : '', " found with ").concat(noConflictsCount == 1 ? 'this' : 'these', ":"), 'color: green; font-size: large');
+      var _data = {};
+
+      for (var _key2 in nodesTested.noConflict) {
+        var _item = nodesTested.noConflict[_key2];
+        _data[_key2] = {
+          'tagName': _item.tagName,
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js:137]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
          "tagName": _item.tagName,
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js [hunk 14]
```diff
@@ -0,0 +1,998 @@
...
+  var _DOCUMENT = {};
+
+  try {
+    if (typeof window !== 'undefined') _WINDOW = window;
+    if (typeof document !== 'undefined') _DOCUMENT = document;
+  } catch (e) {}
+
+  var _ref = _WINDOW.navigator || {},
+      _ref$userAgent = _ref.userAgent,
+      userAgent = _ref$userAgent === void 0 ? '' : _ref$userAgent;
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js:69]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
      userAgent = _ref$userAgent === void 0 ? "" : _ref$userAgent;
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 15]
```diff
@@ -0,0 +1,280 @@
...
+  var _DOCUMENT = {};
+
+  try {
+    if (typeof window !== 'undefined') _WINDOW = window;
+    if (typeof document !== 'undefined') _DOCUMENT = document;
+  } catch (e) {}
+
+  var _ref = _WINDOW.navigator || {},
+      _ref$userAgent = _ref.userAgent,
+      userAgent = _ref$userAgent === void 0 ? '' : _ref$userAgent;
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:18]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
      userAgent = _ref$userAgent === void 0 ? "" : _ref$userAgent;
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js (hunk 4 above)
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/fontawesome.js:162]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var IS_DOM = !!DOCUMENT.documentElement && !!DOCUMENT.head && typeof DOCUMENT.addEventListener === "function" && typeof DOCUMENT.createElement === 'function';
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 16]
```diff
@@ -0,0 +1,280 @@
...
+      ownKeys.forEach(function (key) {
+        _defineProperty(target, key, source[key]);
+      });
+    }
+
+    return target;
+  }
+
+  var w = WINDOW || {};
+  if (!w[NAMESPACE_IDENTIFIER]) w[NAMESPACE_IDENTIFIER] = {};
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:80]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  if (!w[NAMESPACE_IDENTIFIER]) {w[NAMESPACE_IDENTIFIER] = {};}
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js [hunk 17]
```diff
@@ -0,0 +1,998 @@
...
+    if (conflictsCount > 0) {
+      console.info("%cConflict".concat(conflictsCount > 1 ? 's' : '', " found:"), 'color: darkred; font-size: large');
+      var data = {};
+
+      for (var _key in nodesTested.conflict) {
+        var item = nodesTested.conflict[_key];
+        data[_key] = {
+          'tagName': item.tagName,
+          'src/href': item.src || item.href || 'n/a',
+          'innerText excerpt': item.innerText && item.innerText !== '' ? item.innerText.slice(0, 200) + '...' : '(empty)'
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/conflict-detection.js:121]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
          "innerText excerpt": item.innerText && item.innerText !== '' ? item.innerText.slice(0, 200) + '...' : '(empty)'
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js (hunk 8 above)
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/v4-shims.js:29]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  var NAMESPACE_IDENTIFIER = "___FONT_AWESOME___";
```

File: src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js [hunk 18]
```diff
@@ -0,0 +1,280 @@
...
+  if (!w[NAMESPACE_IDENTIFIER].styles) w[NAMESPACE_IDENTIFIER].styles = {};
+  if (!w[NAMESPACE_IDENTIFIER].hooks) w[NAMESPACE_IDENTIFIER].hooks = {};
```
Comment (minhphien) [inline src/PatientChecking/PatientChecking/wwwroot/lib/font-awesome/js/regular.js:82]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/minhphien/kmshcdc-patient-checking/pullRequest?prid=7913498)
```suggestion
  if (!w[NAMESPACE_IDENTIFIER].hooks) {w[NAMESPACE_IDENTIFIER].hooks = {};}
```

File: src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml (hunk 12 above)
Comment (harryul) [reply src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml:84]: the first two @@ is for display @ in razor code block, and the last one is for transition result[1] variable to HTML content. Please review this @vietvo361

**Pull request closed without merging**